### PR TITLE
Removes VERIFY/CHECK macros from the chunk_info test

### PR DESCRIPTION
**Pull request by derobins**
Mixing testhdf5 and h5test macros is a bad idea. Also quiets a couple
of MSVC warnings.

### CODE REVIEW

**Comment by derobins** [comment]
Didn't mean to push all those at once. Fat-fingered commit -a...

File: test/chunk_info.c [hunk 1]
```diff
@@ -179,11 +179,18 @@ verify_get_chunk_info(hid_t dset, hid_t dspace, hsize_t chk_index, hsize_t exp_c
...
+        FAIL_PUTS_ERROR("address cannot be HADDR_UNDEF");
+    if (size != exp_chk_size)
+        FAIL_PUTS_ERROR("unexpected chunk size");
+    if (read_flt_msk != exp_flt_msk)
+        FAIL_PUTS_ERROR("unexpected filter mask");
+    if (out_offset[0] != exp_offset[0])
+        FAIL_PUTS_ERROR("unexpected offset[0]");
+    if (out_offset[1] != exp_offset[1])
+        FAIL_PUTS_ERROR("unexpected offset[1]");
+
```
Comment (bmribler) [inline test/chunk_info.c:193]: What is the reason for this change?  We've been using VERIFY for years and many places, haven't we?  Can VERIFY be modified instead?
